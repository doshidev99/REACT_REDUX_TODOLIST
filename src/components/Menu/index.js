import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import AddTodo from '../../containers/AddTodo';
import VisibleTodoList from '../../containers/VisibleTodoList';
import Footer from '../Footer';

const Menu = () => {
  return (
    <Fragment>
      Hello
      <AddTodo />
      <VisibleTodoList />
      <Footer />
    </Fragment>
  );
};

export default Menu;
