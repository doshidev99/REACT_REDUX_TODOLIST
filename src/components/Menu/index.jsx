import React, { Fragment } from 'react';
import AddTodo from '../../containers/AddTodo';
import VisibleTodoList from '../../containers/VisibleTodoList';
import Footer from '../Footer';
import Header from '../Header';

const Menu = () => {
  return (
    <Fragment>
      <Header />
      <AddTodo />
      <VisibleTodoList />
      <Footer />
    </Fragment>
  );
};

export default Menu;
