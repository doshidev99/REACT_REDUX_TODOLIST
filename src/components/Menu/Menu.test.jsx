/* eslint-disable jsx-a11y/anchor-is-valid */
import React from 'react';
import { shallow } from 'enzyme';
import Menu from './index';

describe('Menu', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Menu debug />);

    expect(component).toMatchSnapshot();
  });
});
