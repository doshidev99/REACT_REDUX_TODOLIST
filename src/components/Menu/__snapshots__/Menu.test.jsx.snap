// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render correctly in "debug" mode 1`] = `
<Fragment>
  <Header />
  <Connect(AddTodo) />
  <Connect(TodoList) />
  <_default />
</Fragment>
`;
